examples/tutorial.ipynb

```diff
@@ -173,26 +162,37 @@
    "source": [
     "#### imagelab.issues\n",
     "\n",
-    "DataFrame assessing each image in your dataset, reporting which issues this image exhibits and a quality score for each type of issue. There is a Boolean column for each issue type, showing whether each image exhibits that type of issue or not. For example,  the rows where the `is_dark_issue` column contains `True` entries correspond to those images that appear too **dark**. For this issue type (and more generally each other type of issue), there is a numeric column `dark_score`, which assess how severe this issue is in each image. These quality scores lie between 0 and 1, where lower values indicate more severe instances of the issue (images which are more dark in this example).  One use-case for `imagelab.issues` is to filter out all images exhibiting  one particular type of issue and rank them by their quality score."
+    "DataFrame assessing each image in your dataset, reporting which issues each image exhibits and a quality score for each type of issue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imagelab.issues.head()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Get all images with `is_blurry_issue=True`"
+    "There is a Boolean column for each issue type, showing whether each image exhibits that type of issue or not. For example,  the rows where the `is_dark_issue` column contains `True`, those rows correspond to images that appear too **dark**.\n",
+    "\n",
+    "For **dark** issue type (and more generally for other types of issues), there is a numeric column `dark_score`, which assesses how severe this issue is in each image. These quality scores lie between 0 and 1, where lower values indicate more severe instances of the issue (images which are more dark in this example).\n",
+    "\n",
+    "One use-case for `imagelab.issues` is to filter out all images exhibiting  one particular type of issue and rank them by their quality score. In the following cell, it is shown how to get all blurry images ranked by their `blurry_score`, note lower scores indicate higher severity."
```

Comment:
```suggestion
    "One use-case for `imagelab.issues` is to filter out all images exhibiting  one particular type of issue and rank them by their quality score. Here's how to get all blurry images ranked by their `blurry_score`, note lower scores indicate higher severity:"
```

Reply:
avoid using passive voice whenever possible, try to keep text as concise as possible